doc/irmin-pack/design/layered_store.md

```diff
@@ -1,23 +1,23 @@
-﻿# Irmin-pack layered-store/GC design document
+﻿# Irmin-Pack Layered Store/GC Design Document
 
 Date: 2022-02-03
 
 ## Introduction
 
 This is a short document to describe the design of a garbage
-collection mechanism for irmin-pack (the variant of Irmin currently
+collection mechanism for `irmin-pack` (the variant of Irmin currently
 used by Tezos).
 
 **TL;DR**
 
-The irmin pack file stores objects. References from one object to
+The `irmin-pack` file stores objects. References from one object to

```

Comment:
```suggestion
A pack file stores Irmin's objects. References from one object to
```
The name "pack file" describes a file which packs multiple objects (so `irmin-pack` got his name from that way of organizing the data on disk). The previous sentence was ambiguous as you noticed, but I'm afraid that "the `irmin-pack` file" reformulation would carry a different meaning than intended... I'm not sure if my suggestion is clearer, let me know if it's still confusing or if you think of a better way to phrase it!

Reply:
Perfect!